src/main.rs

```diff
@@ -32,17 +40,33 @@ struct ComponentCode {
     code: String,
 }
 
-fn handle_request(account_id: &str, path: PathBuf) -> FileList {
+fn handle_request(account_id: &str, path: PathBuf, network: Network) -> FileList {
     let mut components = HashMap::new();
-    get_file_list(&path, account_id, &mut components, String::from(""));
+    get_file_list(&path, account_id, &mut components, String::from(""), network);
     FileList { components }
 }
 
+fn replace_placeholders(code: &str, account_id: &str, network: Network) -> String {
```

Comment:
I _think_ that you can use `&Network` as the type which would prevent you from needing to clone when invoking this function